app.py

```
@@ -106,20 +124,26 @@ def update_frame(frame_index):
                     frame_index = (frame_index + 1) % len(frames)
                     window.after(50, update_frame, frame_index)
-
+                    
                 window.after(0, update_frame, 0)
                 window.deiconify()
 
                 # Close the popup window after the GIF window is displayed
                 popup.destroy()
 
                 window.after(POPUP_DURATION * 1000, window.destroy)
+                logging.info(f"Image window will close after {POPUP_DURATION} seconds.")
```

Ruff

Logging statement uses f-string

(G004)